pkg/image/build.go

```diff
@@ -31,6 +34,8 @@ const bundledSchemaPy = ".cog/schema.py"
 func Build(cfg *config.Config, dir, imageName string, secrets []string, noCache, separateWeights bool, useCudaBaseImage string, progressOutput string, schemaFile string, dockerfileFile string, useCogBaseImage bool) error {
 	console.Infof("Building Docker image from environment in cog.yaml as %s...", imageName)
 
+	ping()
```

Comment:
What do you think about putting this in `cli` instead of `pkg`? If this is intended for interactive consumption by a user, I think that could be a better fit.